src/test/java/calculator/StringCalculatorTest.java

```diff
@@ -0,0 +1,65 @@
+package calculator;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+/**
+ * @author Geonguk Han
+ * @since 2020-02-07
+ */
+class StringCalculatorTest {
+
+    private StringCalculator stringCalculator;
+
+    @BeforeEach
+    void setUp() {
+        stringCalculator = new StringCalculator();
+    }
+
+    @DisplayName("빈 문자열 더하기 테스트")
+    @ParameterizedTest
+    @NullAndEmptySource
+    void add_empty(final String inputValue) {
+        final int result = stringCalculator.add(inputValue);
+        assertThat(result).isEqualTo(0);
+    }
+
+    @DisplayName("값이 한개인 경우 테스트")
+    @ParameterizedTest
+    @ValueSource(strings = {"1"})
+    void add_one(final String inputValue) {
+        final int result = stringCalculator.add(inputValue);
+        assertThat(result).isEqualTo(1);
```

Comment:
`inputValue`를 사용하면 어떨까요?

```suggestion
        assertThat(result).isEqualTo(inputValue);
```

Reply:
inputValue 값이 String 이고, result 결과값이 int 형이라서 isEqualTo 값이 같지가 않다고 나오네요 ! 
`Interger.parseInt(inputValue)` 로 다음과 같이 수정을 생각하고 있습니다.